src/pages/resultPage.js

```diff
@@ -0,0 +1,40 @@
+import { USER_INTERFACE_ID } from '../constants.js';
+import { quizData } from '../data.js';
+import { initWelcomePage } from '../pages/welcomePage.js';
+
+
+export const showResultPage = () => {
+    const userInterface = document.getElementById(USER_INTERFACE_ID);
+    userInterface.innerHTML = '';
+
+    const totalPoints = 15;
```

Comment:
Data like these are better located in the model (data.js) also it's common convention for the variable to be in [screaming snake case](https://en.wikipedia.org/wiki/Snake_case).